pkgs/by-name/mi/miracode/package.nix

```
@@ -0,0 +1,33 @@
+{ stdenv, lib, fetchurl }:
+
+let
+  version = "1.0";
+in
+stdenv.mkDerivation {
+  pname = "miracode";
+  inherit version;
+
+  src = fetchurl {
+    url = "https://github.com/IdreesInc/Miracode/releases/download/v${version}/Miracode.ttf";
+    hash = "sha256-Q+/D/TPlqOt779qYS/dF7ahEd3Mm4a4G+wdHB+Gutmo=";
+  };
+
+  dontUnpack = true;
+  dontConfigure = true;
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/share/fonts/truetype
+    ln -s $src $out/share/fonts/truetype/Miracode.ttf
```

```suggestion
    install -Dm644 $src $out/share/fonts/truetype/Miracode.ttf
```

(No need to mkdir)

Don't use `ln`, as we don't want the final package to depend on `src`